apps/process/views.py

```diff
@@ -1,5 +1,123 @@
 from django.shortcuts import render
+from apps.calc.read_data import read
+from django.contrib.auth.decorators import login_required
+import json
+from .json import NumPyArangeEncoder
+
 
 # Create your views here.
-def index(request):
-    return render(request, "process/index.html")
\ No newline at end of file
+@login_required
+def fromDashboard(request):
+    if request.method == 'POST':
```

Comment:
Besserer:

if not request.method == 'POST':
  -> Hier kein Post also nur template rendern.
  return render.....

-> Hier Post